src/Environment.php

```diff
@@ -854,7 +854,6 @@ private function updateOptionsHash(): void
             self::VERSION,
             (int) $this->debug,
             (int) $this->strictVariables,
-            $this->useYield ? '1' : '0',
```

Comment:
This should be kept as the compiled template is still different in yield mode so we need to invalidate the cache based on this option.

Reply:
Because of the CaptureNode indeed. Updated.

Reply:
technically, also because of the node visitor switching between a deprecation warning and an error (as you would not get the error after enabling the option if the cache was not invalidated)